test/unit/presenters/publishing_api_presenters/detailed_guide_test.rb

```diff
@@ -63,7 +63,7 @@ def present(edition)
           slug: government.slug,
           current: government.current?
         },
-        related_mainstream_content: [],
+        related_mainstream_content: []
```

Comment:
The [styleguide](https://github.com/alphagov/styleguides/blob/master/ruby.md) encourages trailing commas for "clearer diffs with less line noise" (not sure if thats currently in the linter though)